app/models/identifier.js

```diff
@@ -0,0 +1,24 @@
+import DS from 'ember-data';
+import Fragment from 'ember-data-model-fragments/fragment';
+import { validator, buildValidations } from 'ember-cp-validations';
+import { computed } from '@ember/object';
+
+const Validations = buildValidations({
+  identifierType: [
```

Comment:
@mfenner  for example this validation should fail if identifierType is not set and identifier is set. But I doesn't. I don't know if there is something else missing to be configured

Reply:
I'm not sure on this either, the setup looks correct as far as I can tell. My only thoughts were perhaps if the validation was passing but it was just failing to render out the warning.

Reply:
I think i found it. So validations do not work at all for fragments. But the errors are caught with this helper

Reply:
Validations work with fragments, but it is messy. I refactored in January to use the doi-form-errors helper.